scripts/version-validation.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+set -e
+
+if [[ -n $1 ]];
+then
+    if [[ $1 =~ ^(0|[1-9][[:digit:]]*)\.(0|[1-9][[:digit:]]*)\.(0|[1-9][[:digit:]]*)$ ]];
+    then
+        echo "version input is valid";
+    else
+        echo "version input is invalid";
+        exit 1;
+    fi
+else
+    echo "RELEASE_VERSION build parameter is missing";
```

Comment:
`RELEASE_VERSION` var is out of this context - it's Bitrise specific. It's better to print something more generic like "version input data is missing"

Reply:
@imaginaris Yes, I forgot to remove it. I changed the string. Fixed.